extension-dev.code-workspace

```diff
@@ -44,7 +46,17 @@
     "powershell.codeFormatting.whitespaceBetweenParameters": true,
     "powershell.codeFormatting.pipelineIndentationStyle": "IncreaseIndentationForFirstPipeline",
     // Lock the TypeScript SDK path to the version we use
-    "typescript.tsdk": "Client/node_modules/typescript/lib"
+    "typescript.tsdk": "Client/node_modules/typescript/lib",
+    // Code actions like "organize imports" ignore eslint, so we need this here
+    "typescript.format.semicolons": "insert",
+    // Enable ESLint as defaut formatter so quick fixes can be applied directly
+    "eslint.format.enable": true,
+    "[typescript]": {
+      "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+      "editor.formatOnPaste": true,
+      "editor.formatOnSave": true,
+      "editor.formatOnSaveMode": "modificationsIfAvailable"
+    }
```

Comment:
Love it.

Reply:
There will be more on the way, I plan to add some more eslint plugins for mocha (like catching the fact I keep forgetting to remove my .only, etc.), it's a good start though :)